src/test/java/core/basesyntax/strategy/StrategyTest.java

```diff
@@ -0,0 +1,65 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StrategyTest {
+    private static StorageDao dao;
+    private static OperationHandler supplyHandler;
+    private static OperationHandler purchaseHandler;
+    private static OperationHandler balanceHandler;
+    private static Strategy strategy;
+    private static Map<String, OperationHandler> operation;
+
+    @BeforeClass
+    public static void beforeClass() {
+        dao = new StorageDaoImpl();
+        supplyHandler = new SupplyHandlerImpl(dao);
+        purchaseHandler = new PurchaseHandlerImpl(dao);
+        balanceHandler = new BalanceHandler(dao);
+        strategy = new Strategy(dao);
+        operation = new Strategy(dao).getMap();
+    }
+
+    @Test
+    public void get_validOutput_Ok() {
+        OperationHandler expected = purchaseHandler;
+        OperationHandler actual = strategy.get("p");
+        Assert.assertEquals(expected.getClass(), actual.getClass());
```

Comment:
те ж саме варто зробити з рештою, нам не потрібно створювати зайвих об'єктів. Ми можемо оперувати лише класами
```suggestion
        Class<?> expected = PurchaseHandlerImpl.class;
        Class<?> actual = strategy.get("p").getClass();
        Assert.assertEquals(expected, actual);
```